### DISCUSSION TITLE
Is the Microsoft Store no longer a supported source in WingetUI 3.0.0?

### DISCUSSION THREAD

**Discussion by du5tyfinger5** [Devolutions/UniGetUI#1920] [General]
With WingetUI 3.0.0 I no longer get packages from the Microsoft Store when searching in Discover Packages or in the Installed Packages.

![discover_packages](https://github.com/marticliment/WingetUI/assets/118229146/4e8a046e-7d7f-4353-ab69-998758405475)

Edit: I checked and I see the Store set as a source for Winget.

![package_managers](https://github.com/marticliment/WingetUI/assets/118229146/e1b7d8ef-aa9a-4f0e-b919-3bd6efa0a785)

**Comment by du5tyfinger5** [comment]
I installed 2.2.0 in a Sandbox and I cannot search for Store apps in Discover Packages either. I was thinking that I could before.

**Comment by marticliment** [comment]
MS Store packages work for me

**Reply by roke-julian-lockhart** [reply]
> https://github.com/marticliment/UniGetUI/discussions/1920#discussioncomment-8778586

@marticliment, likewise, using https://github.com/marticliment/UniGetUI/releases/download/3.1.1/UniGetUI.Installer.exe on build 26100.1591 of 11 Pro, per the undermentioned:

1.	> ![image](https://github.com/user-attachments/assets/93100445-7ab6-4bb5-85be-7c20079cda4a)

1.	<blockQuote>
	
	```log
	PS C:\Users\rokej> Get-WinGetSource | Format-Table -AutoSize
	
	Name    Argument                                      Type
	----    --------                                      ----
	msstore https://storeedgefd.dsx.mp.microsoft.com/v9.0 Microsoft.Rest
	winget  https://cdn.winget.microsoft.com/cache        Microsoft.PreIndexed.Package
	```
	
	</blockQuote>
